### ISSUE TITLE
Iceberg should be able to manage extermal files

### ISSUE THREAD

**Issue by jecisc** [npasserini/iceberg#805]
I checked all the issues and I found nothing about this one:

Before the release of Iceberg 1.0.0 I think it's important to work on the management of external files with Iceberg.

When I change a Travis, Appveyor, README, script, etc file I always need to commit outside the image then repair my Iceberg repository because it ends up in a broken state. I think it's the most irritating part of Iceberg in the current version. 

It would be really cool if Iceberg would be able to detect changes outside the image and commit them.

**Comment by guillep** [comment]
No, that will never be in 1.0.0 (unless you do it...) because having support for files requires support for editing the files from Pharo also. Maybe for a 1.1 or 1.2, but it should be thought and designed seriously and it takes time.

**Comment by cdlm** [comment]
I often rebase to reorder or merge WIP commits, and actually I've found Iceberg to be quite reliable to that. I'd rather have to switch out of Pharo and use Emacs/Magit for external files than ask the team to waste months on a file editor.

Maybe the easiest compromise is to at least show that there are non-tonel changes in the working copy or in the index? No editing, just something similar to `git status`. But: is the index shared between iceberg and normal git tools?
